tt_metal/llrt/blackhole/bh_hal_active_eth.cpp

```diff
@@ -30,6 +30,7 @@ HalCoreInfoType create_active_eth_mem_map() {
 
     mem_map_bases.resize(utils::underlying_type<HalL1MemAddrType>(HalL1MemAddrType::COUNT));
     mem_map_bases[utils::underlying_type<HalL1MemAddrType>(HalL1MemAddrType::BARRIER)] = MEM_L1_BARRIER;
+    mem_map_bases[utils::underlying_type<HalL1MemAddrType>(HalL1MemAddrType::MAILBOX)] = MEM_MAILBOX_BASE;
```

Comment:
this should be `eth_l1_mem::address_map::ERISC_MEM_MAILBOX_BASE` same for wh active eth